docs/reference/plugins/flutter_plugin.rst

```diff
@@ -0,0 +1,61 @@
+.. _reference-flutter-plugin:
+
+Flutter plugin
+==============
+
+The Flutter plugin simplifies the building of applications that employ the `Flutter
+<https://flutter.dev/>`_ framework.
```

Comment:
```suggestion
<https://flutter.dev/>`_ app framework.
```

Or maybe "UI"? A nitpick, but it might be nice to be more qualitative in what it is.